runtime/common/src/crowdloan/mod.rs

```diff
@@ -49,8 +49,6 @@
 //! the parachain remains active. Users can withdraw their funds once the slot is completed and funds are
 //! returned to the crowdloan account.
 
-pub mod migration;
-
```

Comment:
Why remove this?

Seems fine to expose this module, and keep the migration above.

Reply:
seemed sensible to remove the dead code together with its callsites
anyway, reverted